db/migrate/20191111092601_create_bookmarks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBookmarks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bookmarks do |t|
+      t.integer :user_id
+      t.integer :article_id
```

Comment:
The same thing about using t.references goes for user_id and article_id here. Reference: https://stackoverflow.com/a/39502978/5699251